recipes/pe-configurator/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pe-configurator" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pe-configurator-{{ version }}.tar.gz
+  sha256: 5974cdc5a3d72a5a1493fe8612ac81bfc8401edcf9ca6ef21cc125de67d897a0
+
+build:
+  entry_points:
+    - peconfigurator = peconfigurator.proc_samples:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools >=61.0
+    - setuptools-scm >=3.4.3
+    - wheel
+    - pip
+  run:
+    - python >=3.6
+    - asimov >=0.4.0
+    - lalsuite >=7.14
+    - pandas
+    - pesummary
+    - nbformat
+
+test:
+  imports:
+    - peconfigurator
+  commands:
+    - pip check
+    - peconfigurator --help
+  requires:
+    - pip
+
+about:
+  home: https://git.ligo.org/pe/get_eob_settings/
```

Comment:
That URL looks protected?

Reply:
@dopplershift Thanks for spotting this; I've replaced with a public page.